sdk/appcenter-storage/src/androidTest/java/com/microsoft/appcenter/storage/LocalDocumentStorageAndroidTest.java

```diff
@@ -50,6 +51,7 @@
     private LocalDocumentStorage mLocalDocumentStorage;
 
     private static String mUserTableName = Utils.getUserTableName("123");
+    private static String mReadOnlyTableName = Utils.getTableName(Constants.READONLY, "123");
```

Comment:
Static fields start with s, not m (static vs member).
Also when you can use static final String instead (and in that case use THIS_CASING).

Also add blank lines between fields.